INSTALL.md

```diff
@@ -1221,30 +1224,29 @@ Test OpenSSL
 After a successful build, and before installing, the libraries should
 be tested.  Run:
 
-    $ make test                                      # Unix
-    $ mms test                                       ! OpenVMS
-    $ nmake test                                     # Windows
+    make test                                      # Unix
+    mms test                                       ! OpenVMS
+    nmake test                                     # Windows
 
 **Warning:** you MUST run the tests from an unprivileged account (or disable
 your privileges temporarily if your platform allows it).
 
-If some tests fail, take a look at the [Test Failures](#test-failures)
-subsection of the [Troubleshooting](#troubleshooting) section.
+See the file [test/README.md](test/README.md) for further details.
 
 Install OpenSSL
 ---------------
 
 If everything tests ok, install OpenSSL with
 
-    $ make install                                   # Unix
-    $ mms install                                    ! OpenVMS
-    $ nmake install                                  # Windows
+    make install                                   # Unix
+    mms install                                    ! OpenVMS
+    nmake install                                  # Windows
 
 Note that in order to perform the install step above you need to have
 appropriate permissions to write to the installation directory.
 
 The above commands will install all the software components in this
-directory tree under PREFIX (the directory given with `--prefix` or
+directory tree under <PREFIX> (the directory given with `--prefix` or
```

Comment:
Maybe backticks around <PREFIX> or there is the risk some markdown renderer might treat it as inline html

Reply:
Good thought - done.